### DISCUSSION TITLE
git staging issue. how can i fix it.

### DISCUSSION THREAD

**Discussion by yusuphoo** [Cyfrin/foundry-full-course-cu#1531] [Q&A]
###  why sometimes these stuffs stuck on my git. How can i avoid it.
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/103340487/1a064070-c00c-498f-9651-83f9da426a8c)

**Comment by EngrPips** [comment]
Never use `wsl` before, kinda always hard for me to figure out what's going when people that use it have issues using it. Does it crash your machine or what ?

**Reply by yusuphoo** [reply]
nope. its trying to push all these unneccery stuffs to github.

**Reply by EngrPips** [reply]
So did it successfully push it ?

**Reply by yusuphoo** [reply]
yes! I some how did it. I created another folder inside the current folder and move my project to it. and all the stuffs gone.

**Reply by EngrPips** [reply]
Awesome, i am glad you figured it out.

**Reply by yusuphoo** [reply]
drop ur social media handle lets connect.

**Reply by EngrPips** [reply]
Discord @engr.pips, looking forward to hear from you
